README.md

```diff
@@ -24,104 +18,68 @@ of active locales is:
 - `zh` (`zh-CN` and `zh-TW`)
 - `es`
 
-If you want to just find a task and jump in, search by the labels `l10n-fr`,
-`l10n-ja`, `l10n-ko`, `l10n-pt-br`, `l10n-ru`, `l10n-zh` and `l10n-es` in this repo's
-[issues list](https://github.com/mdn/translated-content/issues),
-or the main [content repo issues](https://github.com/mdn/content/issues).
-
----
+If you want to just find a task and jump in, search by the labels `l10n-fr`, `l10n-ja`, `l10n-ko`, `l10n-pt-br`, `l10n-ru`, `l10n-zh` and `l10n-es` in this repo's [issues list](https://github.com/mdn/translated-content/issues), or the main [content repo issues](https://github.com/mdn/content/issues).
 
 ## Code of Conduct
 
-Everyone participating in this project is expected to follow our
-[Code of Conduct](CODE_OF_CONDUCT.md).
+Everyone participating in this project is expected to follow our [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## License
 
-When contributing to the content you agree to license your contributions
-according to [our license](LICENSE.md).
-
----
+When contributing to the content you agree to license your contributions according to [our license](LICENSE.md).
 
 ## Making contributions
 
 A good place to learn about general guidelines for contributing to
```

Comment:
One other comment, this line is dangling :)

Reply:
Thank you, I missed this one 😅